Automattic/jetpack
# Untangling: Show WP.com themes banner

Fixes https://github.com/Automattic/dotcom-forge/issues/6475
Fixes https://github.com/Automattic/dotcom-forge/issues/6477

## Proposed changes:
This PR displays a banner on untangled Atomic sites before the theme browser that links to the WP.com Theme Showcase.

<img width="1278" alt="Screenshot 2024-04-09 at 16 18 04" src="https://github.com/Automattic/jetpack/assets/1233880/c63c988f-0db8-473d-b06b-648fbeec9e51">


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- If you don't have an untangled Atomic site:
  - Create or pick a regular Atomic site
  - Go to wordpress.com/hosting-config and select your Atomic site
  - Change the admin interface to wp-admin
  - Go to `<SITE_DOMAIN>/_cli` and run this command: `wp option update wpcom_classic_early_release 1`
- Apply these changes to your untangled Atomic site:
  - Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/)
  - Go to `/wp-admin/admin.php?page=jetpack-beta&plugin=jetpack-mu-wpcom-plugin`
  - Activate this branch
- Go to `/wp-admin/theme-install.php`
- Make sure the WP.com themes banner show up
- Make sure it looks good
- Make sure it links to the WP.com theme showcase
- Try different browser sizes and check the banner responsiveness
- Make sure the banner does not show up in other pages
